src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import classNames from 'classnames';
+import { FilterType } from '../../helpers/filterTodos';
+
+interface Props {
+  activeTodosCount: number,
+  hasCompletedTodo: boolean,
+  filterType: FilterType;
+  setFilter: React.Dispatch<React.SetStateAction<FilterType>>;
+}
+
+export const Footer: React.FC<Props> = ({
+  activeTodosCount,
+  hasCompletedTodo,
+  filterType,
+  setFilter,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodosCount} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter">
+        <a
```

Comment:
not blocker: check how you can reduce your code with array constant and `.map`

Your have a lot of similar blocks with small differences like `FilterType`, `label` and `href`